Detectors/EMCAL/simulation/include/EMCALSimulation/MCLabel.h

```diff
@@ -0,0 +1,39 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+
+// Declaration of a transient MC label class for TOF
+
+#ifndef ALICEO2_EMCAL_MCLABEL_H_
+#define ALICEO2_EMCAL_MCLABEL_H_
+
+#include "SimulationDataFormat/MCCompLabel.h"
+
+namespace o2
+{
+namespace EMCAL
+{
+class MCLabel : public o2::MCCompLabel
```

Comment:
so do I understand correctly that this label is only used temporarily and serves to backup the event time of the eventID contributing to a digit?

Reply:
The labels associated with each Digit are sorted according to the event time, which is why we are storing that quantity in this class. Again, I based this on the TOF code, which sorts based on the TDC value. @mfasDa said that we want to be sorting on the event time.